apps/site/package.json

```diff
@@ -2,37 +2,47 @@
   "name": "@node-core/website",
   "type": "module",
   "scripts": {
-    "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "dev": "cross-env NODE_NO_WARNINGS=1 next dev",
-    "serve": "npm run dev",
     "build": "cross-env NODE_NO_WARNINGS=1 next build --turbopack",
-    "start": "cross-env NODE_NO_WARNINGS=1 next start",
-    "deploy": "cross-env NEXT_PUBLIC_STATIC_EXPORT=true NODE_NO_WARNINGS=1 next build",
     "check-types": "tsc --noEmit",
-    "lint:js": "eslint \"**/*.{js,mjs,ts,tsx}\"",
-    "lint:snippets": "node ./scripts/lint-snippets/index.mjs",
-    "lint:md": "eslint \"**/*.md?(x)\" --cache --cache-strategy=content --cache-location=.eslintmdcache",
-    "lint:css": "stylelint \"**/*.css\" --allow-empty-input --cache --cache-strategy=content --cache-location=.stylelintcache",
+    "deploy": "cross-env NEXT_PUBLIC_STATIC_EXPORT=true NODE_NO_WARNINGS=1 next build",
+    "dev": "cross-env NODE_NO_WARNINGS=1 next dev",
     "lint": "turbo run lint:md lint:snippets lint:js lint:css",
+    "lint:css": "stylelint \"**/*.css\" --allow-empty-input --cache --cache-strategy=content --cache-location=.stylelintcache",
     "lint:fix": "turbo run lint:md lint:js lint:css --no-cache -- --fix",
+    "lint:js": "eslint \"**/*.{js,mjs,ts,tsx}\"",
+    "lint:md": "eslint \"**/*.md?(x)\" --cache --cache-strategy=content --cache-location=.eslintmdcache",
+    "lint:snippets": "node ./scripts/lint-snippets/index.mjs",
+    "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
+    "serve": "pnpm dev",
+    "start": "cross-env NODE_NO_WARNINGS=1 next start",
     "sync-orama": "node ./scripts/orama-search/sync-orama-cloud.mjs",
+    "test": "turbo test:unit",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 node --experimental-test-coverage --test-coverage-exclude=**/*.test.* --experimental-test-module-mocks --enable-source-maps --import=global-jsdom/register --import=tsx --import=tests/setup.jsx --test **/*.test.*",
-    "test:unit:watch": "cross-env NODE_OPTIONS=\"--watch\" npm run test:unit",
-    "test": "turbo test:unit"
+    "test:unit:watch": "cross-env NODE_OPTIONS=\"--watch\" pnpm test:unit"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api-logs": "~0.200.0",
+    "@opentelemetry/instrumentation": "~0.200.0",
+    "@opentelemetry/resources": "~1.30.1",
+    "@opentelemetry/sdk-logs": "~0.200.0"
   },
```

Comment:
How can the site possibly have peer dependencies? We're not distributing/consuming it elsewhere?

Reply:
I'll move them back to regular deps

Reply:
They appear to be new -- are we using otel now suddenly? That seems like a change that should be landed in a separate PR?

Reply:
They are peer dependencies of `@vercel/otel`